src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -216,197 +198,274 @@
 
-                    <ListView ItemsSource="{x:Bind Path=ViewModel.Plugins, Mode=OneWay}"
-                              IsItemClickEnabled="False"
-                              SelectionMode="None"
-                              x:Name="PluginsListView">
-                        <ListView.ItemTemplate>
-                            <DataTemplate x:DataType="ViewModels:PowerLauncherPluginViewModel" x:DefaultBindMode="OneWay">
+                    <ItemsControl
+                        x:Name="PluginsListView"
+                        ItemsSource="{x:Bind Path=ViewModel.Plugins, Mode=OneWay}">
+                        <ItemsControl.ItemsPanel>
+                            <ItemsPanelTemplate>
+                                <StackPanel Spacing="2" />
+                            </ItemsPanelTemplate>
+                        </ItemsControl.ItemsPanel>
+                        <ItemsControl.ItemTemplate>
+                            <DataTemplate
+                                x:DataType="ViewModels:PowerLauncherPluginViewModel"
+                                x:DefaultBindMode="OneWay">
                                 <Grid>
-                                    <controls:SettingExpander>
-                                        <controls:SettingExpander.Header>
-                                            <controls:Setting Header="{x:Bind Path=Name}" Description="{x:Bind Description}" Style="{StaticResource ExpanderHeaderSettingStyle}">
-                                                <controls:Setting.Icon>
-                                                    <Image Source="{x:Bind IconPath}"
-                                                           Width="20"
-                                                           Height="20" />
-                                                </controls:Setting.Icon>
-                                                <controls:Setting.ActionContent>
-                                                    <StackPanel Orientation="Horizontal" Spacing="16" >
-
-                                                        <!-- todo(Stefan): InfoBadge not available
+                                    <labs:SettingsExpander
+                                        Description="{x:Bind Description}"
+                                        Header="{x:Bind Path=Name}">
+                                        <labs:SettingsExpander.HeaderIcon>
+                                            <BitmapIcon UriSource="{x:Bind IconPath}" />
+                                        </labs:SettingsExpander.HeaderIcon>
+                                        <StackPanel
+                                            Orientation="Horizontal"
+                                            Spacing="16">
+                                            <!-- todo(Stefan): InfoBadge not available
                                                         <InfoBadge AutomationProperties.AccessibilityView="Raw"
```

Comment:
@stefansjfw does this comment mean the InfoBar can/should be enabled?

Reply:
The InfoBar.. or InfoBadge :)? InfoBadge is back in WindowsAppSDK 1.2 - we can remove the border and uncomment the InfoBadge when we upgrade (1.2 stable was released last week 👍)

Reply:
I think there should be a **Bar** here, the **InfoBadge** is just about numbers:
![InfoBadge](https://learn.microsoft.com/en-us/windows/apps/design/controls/images/infobadge/infobadge-example-1.png)

Reply:
What's the actual _purpose_ here again? Because an InfoBar is for numbers, but the replacement code uses `InfoBarErrorIconGlyph`..?

Reply:
P.S.

### Is this the right control?
An [info badge](https://learn.microsoft.com/en-us/windows/apps/design/controls/info-badge) should be used when you want to bring the user's focus to a certain area of your app in an unintrusive way. When an info badge appears, it is meant to bring focus to an area and then let the user get back into their flow, giving them the choice of whether or not to look into the details of why the info badge appeared. Info badges should only represent messages that are dismissible and non-permanent – an info badge should have specific rules as to when it can appear, disappear, and change.

Examples of appropriate info badge usage:

- (...)
- To indicate that there might be an issue with an item on a certain page **that does not block the app from functioning.**